divio_cli/cli.py

```diff
@@ -106,9 +106,27 @@ def login(ctx, token, check):
 
 
 @cli.group()
-def project():
+@click.option(
+    '--id', 'identifier',
+    default=None, type=int,
+    help='Project.id to use for project commands. '
+         'Defaults to the project in the current directory using the .aldryn '
+         'file.'
+)
+@click.pass_obj
+def project(obj, identifier):
```

Comment:
It would be nice if we could show a little more user-friendly message than just printing the project id